code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -728,9 +817,37 @@
 	use_message = "The eggs begin to twitch..."
 	used_message = "The cluster already hatched."
 	failure_message = "<B>...but soon settles again. Guess they weren't ready to hatch after all.</B>"
+	limiteduses = TRUE
 
 /obj/item/weapon/guardiancreator/biological/choose
 	random = FALSE
+	limiteduses = TRUE
+
+/obj/item/weapon/guardiancreator/standarrow
+	name = "Stand Arrow"
+	desc = "A mysterious arrow capable of granting great power. Be carfeul, there is a chance it won't take to you..."
```

Comment:
"A mysterious arrow capable of granting great power. Be carfeul, there is a chance it won't take to you..."
spell check please.